src/components/expanded-state/chart/ChartExpandedStateHeader.js

```diff
@@ -13,36 +11,18 @@ import {
   ChartPriceLabel,
 } from './chart-data-labels';
 import { convertAmountToNativeDisplay } from '@rainbow-me/helpers/utilities';
-import { useAccountSettings, useBooleanState } from '@rainbow-me/hooks';
+import { useAccountSettings } from '@rainbow-me/hooks';
 import { padding } from '@rainbow-me/styles';
-
-const { call, cond, onChange, useCode } = Animated;
+import styled from 'rainbowed-components';
 
 const noPriceData = 'No price data';
 
 const Container = styled(ColumnWithMargins).attrs({
   margin: 12,
   marginTop: android ? -10 : 0,
-})`
-  ${({ showChart }) => padding(0, 19, showChart ? (android ? 15 : 30) : 0)};
-`;
-
-function useTabularNumsWhileScrubbing(isScrubbing) {
-  const [tabularNums, enable, disable] = useBooleanState();
-  // Only enable tabularNums on the price label when the user is scrubbing
-  // because we are obnoxiously into details
-  useCode(
-    useCallbackOne(
-      () =>
-        onChange(
-          isScrubbing,
-          cond(isScrubbing, call([], enable), call([], disable))
-        ),
-      [disable, enable, isScrubbing]
-    )
-  );
-  return tabularNums;
-}
+})(({ showChart }) => ({
```

Comment:
@terrysahaidak those changes are unrel. We can merge this here unless you prefer to make this in the other PR